docs/styleguide/scripts/main.js

```diff
@@ -1,5 +1,13 @@
-/**
- * Created by d063436 on 5/21/17.
- */
-console.log('Just for js test purpose');
 
+$(document).ready(function(){
+    var primaryNavigationDropdown = $('.yaas-signature-nav__list');
+    $('.primary-nav--toggle').on('click', function(){
```

Comment:
Here you can pick up the `data-js="menu-toggle"` attribute to initialize the JS ...
```
$('[data-js="menu-toggle"]')
```
And then operate on the ARIA attributes ...
* You can find the target list from the `aria-controls` attribute (so you don't have to hardcode that in the JS)
* On click, change  `aria-expanded` on the trigger element — `$(this)` — to true
* Then on the target element, change `aria-hidden` to false
Of course, reverse when closing.

This way you aren't hardcoding a single class and you get accessibility for free.